### PR TITLE
Fix for workset queues bug

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

**Review by ssjunnebo — APPROVED** [review]
👍

File: status/worksets.py [hunk 1]
```diff
@@ -325,10 +325,10 @@ def get(self):
...
         for method in methods:
             pools[method] ={}
-            query = ('select art.artifactid, art.name, st.lastmodifieddate, st.generatedbyid '
-                            'from artifact art, stagetransition st where art.artifactid=st.artifactid and '
-                            'st.stageid in (select stageid from stage where stepid={}) and '
-                            'st.completedbyid is null and st.workflowrunid>0;'.format(queues[method]))
+            query = ("select art.artifactid, art.name, st.lastmodifieddate, st.generatedbyid "
+                            "from artifact art, stagetransition st where art.artifactid=st.artifactid and "
+                            "st.stageid in (select stageid from stage where stepid={}) and "
+                            "st.completedbyid is null and st.workflowrunid>0 and art.name not in ('lambda DNA');".format(queues[method]))
```
Comment (chuan-wang) [inline status/worksets.py:331]: Is `lambda DNA` the only possible control here?

Comment (aanil) [reply status/worksets.py:331]: I'm not sure, I was hoping you could clarify. It is the only one that has come up in this particular step until now. For the qPCR step we have this list `[ 'AM7852', 'E.Coli genDNA', 'Endogenous Positive Control', 'Exogenous Positive Control', 'Human Brain Reference RNA', 'lambda DNA', 'mQ Negative Control', 'NA10860', 'NA11992', 'NA11993', 'NA12878', 'NA12891', 'NA12892', 'No Amplification Control', 'No Reverse Transcriptase Control', 'No Template Control', 'PhiX v3', 'Universal Human Reference RNA', 'lambda DNA (qPCR)']`

**Review by chuan-wang — APPROVED** [review]
👍

**Pull request merged**